www/base/src/components/BuildSummary/BuildSummary.tsx

```diff
@@ -141,14 +141,17 @@ const BuildSummaryStepLine = observer(({build, step, logs, parentFullDisplay}: B
     return null;
   }
 
+  const renderAmount = (count: number, what: string) => {
+    return `${count} ${what}${count === 1 ? "" : "s"}`
+  }
+
   const maybeRenderPendingBuildCount = () => {
     if (stepInfo.buildrequests.length === 0) {
       return null;
     }
     return (
-      <span>
-        {stepInfo.builds.length} builds,
-        {stepInfo.buildrequests.length - stepInfo.builds.length} pending builds
+      <span className="bb-build-request-counter">
+        {renderAmount(stepInfo.buildrequests.length, "build")}, {renderAmount(stepInfo.buildrequests.length - stepInfo.builds.length, "pending build")}
```

Comment:
Please break this long line into two. HTML collapses spaces, so the displayed output won't change.

Reply:
I tried that, but if I break it over two lines it doesn't have a space, but it has one if it is in one line.